cfpb/cfgov-refresh
# Fixes DF-2349 - updates deputy director to Meredith Fuchs

Updates deputy director to Meredith Fuchs.

## Additions

- Added Meredith Fuchs to Leadership calendar filter.

## Changes

- Updated Deputy Director information to Meredith Fuchs.
- Updated `/about-rich-cordray/` URL to `/about-director/`.
- Updated `/about-meredith-fuchs/` URL to `/about-deputy-director/`.
- Normalized director and deputy director photos to be format `NAME-WxH.jpg`.
- Fixed gulp copy task that was missing copying PDFs in subdirectories.
- Fixed `figure` element that was inside a `p`.

## Testing

- Should be Meredith Fuchs on `/the-bureau/`.
- Should be Meredith Fuchs on `/the-bureau/about-deputy-director/`.
- New URL `/the-bureau/about-director/`
- Visit `/newsroom/press-resources/`
- Visit `/about-us/`

## Review

- @sebworks 
- @jimmynotjim 
- @duelj 

## Screenshots

![screen shot 2015-08-17 at 1 40 59 pm](https://cloud.githubusercontent.com/assets/704760/9311754/8479912c-44e6-11e5-95dd-c0a8a0b3134f.png)

![screen shot 2015-08-17 at 1 48 19 pm](https://cloud.githubusercontent.com/assets/704760/9311774/a2c9b6fc-44e6-11e5-8fb5-bad941f82baa.png)

![screen shot 2015-08-17 at 1 48 50 pm](https://cloud.githubusercontent.com/assets/704760/9311787/b53d34bc-44e6-11e5-9867-1bc72504a5f6.png)

![screen shot 2015-08-17 at 1 49 25 pm](https://cloud.githubusercontent.com/assets/704760/9311803/caff3354-44e6-11e5-910c-f2a12d211137.png)


## Todos

- There are some slight differences in image sizes because I used our existing media image sizing classes. The round images with accompanying link list should be turned into a macro that allows the same code to generate the media image blocks on `/newsroom/press-resources/`, `/the-bureau/`, `/the-bureau/about-deputy-director/`, and `/the-bureau/about-director/`. These currently have slightly different behaviors across responsive sizes. See [GHE]/flapjack/Modules-V1/issues/5
- The `float-media` and `rwd-crop` classes should be incorporated into the media image structure in `media.less`.
